### ISSUE TITLE
Docs gen: Config options in mixin is not getting generated

### ISSUE THREAD

**Issue by suyograo** [elastic/logstash#3616]
Tried generating docs for https://github.com/logstash-plugins/logstash-input-jdbc which has configs in a mixin file.. https://github.com/logstash-plugins/logstash-input-jdbc/blob/master/lib/logstash/plugin_mixins/jdbc.rb 

The configs get pulled to the asciidoc, but the comments associated with it is not getting rendered.

Steps:

```
bin/plugin update logstash-input-jdbc
bin/logstash docgen -o /Users/suyog/ws/elastic/logstash-docs/docs/plugins ./vendor/bundle/jruby/1.9/gems/logstash-input-jdbc-1.0.0/lib/logstash/inputs/jdbc.rb
```

**Comment by ph** [comment]
@suyograo this is fixed in my refactor of the doc generation, will try to push my pr this week :)

**Comment by ph** [comment]
This is a really complex thing, my old branch was working because it used ruby interpretter code to walk the accessor three, I did some test with that and this approch can be fragile when you have  multiple layer of includes (mixin)

**Comment by ph** [comment]
Lets take the case of the rabbitmq plugins, they all use the same config mixin https://github.com/logstash-plugins/logstash-mixin-rabbitmq_connection,

If we look at the documentation page all the fields show ups.

**Comment by ph** [comment]
OK, if you use the specially named namespace `PluginsMixin` the parser pick it up.

**Comment by suyograo** [comment]
Will be fixed in #5715
